pmlc/dialect/tile/ir/ops.cc

```diff
@@ -20,6 +20,12 @@ namespace pmlc::dialect::tile {
 
 using llvm::SmallVector;
 
+LogicalResult ArgSortOp::materializeOperands(OpBuilder &builder) {
+  Operation *op = getOperation();
+  return tile::materializeOperands(builder, op,
+                                   op->getOpOperands().take_front());
```

Comment:
Why do we need `take_front()` here? Not even sure we need a custom `materializeOperands` function for this op.